src/common/components/Checkboxes/Checkboxes.tsx

```diff
@@ -0,0 +1,67 @@
+import { Box, Checkbox, useCheckboxGroup, VStack } from '@chakra-ui/react';
+import { useEffect } from 'react';
+
+import { CheckedIcon, UncheckedIcon } from '../Icons';
+
+export interface ICheckbox {
+  name: string;
+  value: string;
+  icon?: React.ReactNode;
+}
+
+interface CheckboxesProps {
+  items: ICheckbox[];
+  defaultValues: string[];
+  onChange: (value: (string | number)[]) => void;
+}
+
+/**
+ *
+ * values would be array
+ *
+ */
+
+export const Checkboxes = ({ items = [], defaultValues = [], onChange }: CheckboxesProps) => {
+  const { value, getCheckboxProps } = useCheckboxGroup({
+    defaultValue: defaultValues,
+  });
+
+  useEffect(() => {
+    onChange(value);
+  }, [value.length]);
```

Comment:
I think we need to include `onChange` to the `useEffect` dependency because we're using it inside that to ignore the ESlint warning message

```suggestion
  }, [onChange, value.length]);
```

Reply:
yeah onChange was previous an inline function so useEffect kept looping! but i've fixed it!